Dockerfile

```diff
@@ -3,7 +3,7 @@ WORKDIR /build
 COPY . .
 RUN make gobuild
 
-FROM registry.access.redhat.com/ubi8-minimal
+FROM registry.access.redhat.com/ubi8-minimal:8.8
```

Comment:
Depends on if it's more worthwhile to automate it. Specifying version is best practice but I see the worth in using `latest` in this case. Me and @dwelch0 are looking into ways we can trigger a new build without PR, since moving from 8.7 (last build from quay) is ultimately what we are trying to propose here.

Reply:
@kwilczynski, Jonah was following my guidance on this one. I'm a little back and forth on it still because avoiding `latest` has been conditioned into me :sweat_smile: 

But concede that there's likely more risk associated with forgetting to bump the version periodically than to run into a problem when pulling latest.

Reply:
@jonahbrawley we'll trigger a new build within ci.ext and that should suffice :)